admiral.Rproj

```diff
@@ -19,5 +19,6 @@ LineEndingConversion: Posix
 BuildType: Package
 PackageUseDevtools: Yes
 PackageInstallArgs: --no-multiarch --with-keep.source
+PackageRoxygenize: rd,collate,namespace
```

Comment:
Just curious, what does this do?

Reply:
Oh, this is just an option to use ⌘+D to document the RD, collate, and update NAMESPACE. Just a lil dev tool that is handy! If someone doesn't use the RStudio IDE or prefers not to use the tool, their experience is unchanged.